recipe/meta.yaml

```diff
@@ -106,7 +108,16 @@ requirements:
 {% set tests_to_skip = tests_to_skip + " or test_thresholding or test_mpl_imshow" %}  # [win64]
 # 2022/03/18: scikit-image 0.19.2 tests "future/graph/tests/test_rag.py" failed on win and unix
 {% set tests_to_skip = tests_to_skip + " or test_cut_normalized or test_ncut_stable_subgraph or test_reproducibility" %}
-
+# 2023/4/4: scikit-image 0.20.0 tests "test_analytical_moments_calculation"  https://github.com/scikit-image/scikit-image/issues/6865
+{% set tests_to_skip = tests_to_skip + " or test_analytical_moments_calculation" %}
+# The set of tests below are skipped for successful build.
+# There is a library upstream that is missing that causes these
+# tests to fail.
+{% set tests_to_skip = tests_to_skip + " or test_rolling_ball" %}
```

Comment:
Is it on a specific architecture? (the test passes on arm64..). Which library is missing?

Reply:
These tests were explicitly failing for win-64. After analyzing the build failures with @sumit0190, it is highly suspected that there are some broken tests upstream that haven't been fixed as of yet. This is also the case with the current `scikit` 0.19.3 as it has some of the same test failures. Unfortunately to my knowledge, I don't think there's much that we can do regarding this at the moment.

Reply:
Ok, at least can you add `# [win64]` after each skip?

Reply:
Absolutely!